deps/wazuh_testing/wazuh_testing/qa_docs/lib/config.py

```diff
@@ -32,153 +36,194 @@ def __init__(self, *args):
                 self.ignore_paths.append(os.path.join(self.project_path, path))
 
-    def _read_module_fields(self):
+    def __read_module_fields(self):
         """
         brief: Reads from the config file the optional and mandatory fields for the test module.
         """
+        Config.LOGGER.debug('Reading mandatory and optional module fields from config file')
+
         if not 'Module' in self._config_data['Output fields']:
-            logging.error("Config output module fields is missing")
-            raise Exception("Config output module fields is missing")
+            raise QAValueError('Config output module fields is missing', Config.LOGGER.error)
+
         module_fields = self._config_data['Output fields']['Module']
+
         if not 'Mandatory' in module_fields and not 'Optional' in module_fields:
-            logging.error("Config output module fields are empty")
-            raise Exception("Config output module fields are empty")
+            raise QAValueError('Config output module fields are empty', Config.LOGGER.error)
+
         if 'Mandatory' in module_fields:
             self.module_fields.mandatory = module_fields['Mandatory']
+
         if 'Optional' in module_fields:
             self.module_fields.optional = module_fields['Optional']
 
-    def _read_test_fields(self):
+    def __read_test_fields(self):
         """
         brief: Reads from the config file the optional and mandatory fields for the test functions.
         """
+        Config.LOGGER.debug('Reading mandatory and optional test fields from config file')
+
         if not 'Test' in self._config_data['Output fields']:
-            logging.error("Config output test fields is missing")
-            raise Exception("Config output test fields is missing")
+            raise QAValueError('Config output test fields is missing', Config.LOGGER.error)
+    
         test_fields = self._config_data['Output fields']['Test']
+
         if not 'Mandatory' in test_fields and not 'Optional' in test_fields:
-            logging.error("Config output test fields are empty")
-            raise Exception("Config output test fields are empty")
+            raise QAValueError('Config output test fields are empty', Config.LOGGER.error)
+
         if 'Mandatory' in test_fields:
             self.test_fields.mandatory = test_fields['Mandatory']
+
         if 'Optional' in test_fields:
             self.test_fields.optional = test_fields['Optional']
 
-    def _read_output_fields(self):
+    def __read_output_fields(self):
         """
         brief: Reads all the mandatory and optional fields.
         """
         if not 'Output fields' in self._config_data:
-            logging.error("Config output fields is missing")
-            raise Exception("Config output fields is missing")
-        self._read_module_fields()
-        self._read_test_fields()
+            raise QAValueError('Config output fields is missing', Config.LOGGER.error)
```

Comment:
```suggestion
            raise QAValueError('Config output fields are missing', Config.LOGGER.error)
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523